src/sources/dom_blockers.ts

```diff
@@ -356,6 +356,8 @@ export async function getBlockedSelectors<T extends string>(selectors: readonly
   }
   d.body.appendChild(root)
 
+  await releaseEventLoop()
```

Comment:
Looks like this may cause the elements to be displayed on the page for a moment. Probably, `root` should be hidden with CSS such way that `offsetParent` keeps working.

Reply:
We can skip this source to eliminate the risk, considering low profit of this optimisation

Reply:
Sounds good

Reply:
The other DOM sources may have the same issues. `visibility: hidden` will suffice to solve them.

Reply:
Ok, I'll check